### PR TITLE
feat: add publish-operators-for-e2e-tests workflow

### PR DESCRIPTION
**Pull request by MatousJobanek**
similar as is already in host & member operator repos

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

File: .github/workflows/publish-operators-for-e2e-tests.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+env:
+  GOPATH: /tmp/go
+  GO_VERSION: 1.16.x
+
+jobs:
+  binary:
+    name: Build & push operator bundles for e2e tests
+
+    runs-on: ubuntu-18.04
```
Comment (xcoulon) [inline .github/workflows/publish-operators-for-e2e-tests.yml:13]: just curious, what about running on Ubuntu 20.04 (see https://github.com/codeready-toolchain/host-operator/pull/470#discussion_r655521712)

Comment (MatousJobanek) [reply .github/workflows/publish-operators-for-e2e-tests.yml:13]: I'm not sure what is the current status but at that time, when I had to set the specific version, we weren't able to use the latest nor 20.04 - there was a bug in a specific version of Podman that was automatically installed in Ubuntu via the used channel.

**Pull request merged**